hector_path_follower/src/hector_path_follower.cpp

```diff
@@ -317,15 +317,15 @@ namespace pose_follower {
     transformed_plan.clear();
 
     try{
-      if (!global_plan.size() > 0)
+      if (!(global_plan.size() > 0))
```

Comment:
This was definitely a bug, even if in this special case the implicit conversion from the boolean expression `!global_plan.size()` to an integer (which is 0 or 1) leads to the same result when compared against 0 as the intended expression.

Even better:
```cpp
if (global_plan.empty())
{
  // ...
}
```